src/main/java/uk/gov/hmcts/reform/professionalapi/util/OrganisationTypeConstants.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.professionalapi.util;
+
+public class OrganisationTypeConstants {
+    public static final String SOLICITOR_ORG = "SOLICITOR-ORG";
+    public static final String LOCAL_AUTHORITY_ORG = "LOCAL-AUTHORITY-ORG";
+    public static final String OTHER_ORG = "OTHER-ORG";
+    public static final String OGD_DWP_ORG = "OGD-DWP-ORG";
+    public static final String OGD_HO_ORG = "OGD-HO-ORG";
+    public static final String OGD_OTHER_ORG = "OGD-OTHER-ORG";
+
+    private OrganisationTypeConstants() {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
```

Comment:
why we need this exception here?

Reply:
Make the class final and declare the private constructor

Reply:
I can leave an empty ctor but I thought it would be better to fail if someone tried to invoke this